studio/pages/api/props/project/[ref]/settings.ts

```diff
@@ -59,6 +59,7 @@ const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
           db_schema: 'public',
           endpoint: PROJECT_ENDPOINT,
           realtime_enabled: true,
+          protocol: PROJECT_ENDPOINT_PROTOCOL
```

Comment:
Hmm we try to keep this static api response symmetric with what our platform api returns. This change would make the static api response different from that of platform...

Reply:
Yes I know, but the protocol is ignored when using the platform.
Maybe should I add a comment explaining why it is needed?

Reply:
Yup, let's add a comment for now and proceed with this change.

```suggestion
          // TODO: update platform api to return this field
          protocol: PROJECT_ENDPOINT_PROTOCOL
```

@inian do you have any strong opinions about adding the protocol field to middleware api?